agent/config/config.go

```diff
@@ -212,6 +212,8 @@ func EnvironmentConfig() Config {
 	updateDownloadDir := os.Getenv("ECS_UPDATE_DOWNLOAD_DIR")
 	updatesEnabled := utils.ParseBool(os.Getenv("ECS_UPDATES_ENABLED"), false)
 
+	disableMetrics := utils.ParseBool(os.Getenv("ECS_DISABLE_METRICS"), false)
```

Comment:
Technically it's already default false since that's the zero value; if you need to distinguish between false and something else you would use a pointer.

Agreed with adding it to the defaultconfig though for clarity / consistency.